robocop/checkers/lengths.py

```diff
@@ -323,17 +323,7 @@ def check_if_empty(self, node):
     def visit_SettingSection(self, node):  # noqa
         self.check_if_empty(node)
 
-    def visit_VariableSection(self, node):  # noqa
-        self.check_if_empty(node)
-
-    def visit_TestCaseSection(self, node):  # noqa
-        self.check_if_empty(node)
-
-    def visit_KeywordSection(self, node):  # noqa
-        self.check_if_empty(node)
-
-    def visit_CommentSection(self, node):  # noqa
-        self.check_if_empty(node)
+    visit_VariableSection = visit_TestCaseSection = visit_KeywordSection = visit_CommentSection = visit_SettingSection
 
 
 class NumberOfReturnedArgsChecker(VisitorChecker):
```

Comment:
Now when I see this checker I realize we need to update it to work with new RETURN statement (can you create issue?).

Reply:
I will do it.